feat(konnect): Document org role FAQ and TGW limitation (#5229)
Kong/developer.konghq.com: app/dedicated-cloud-gateways/transit-gateways.md

    1. Enable **Allow external accounts**, choose **AWS Account**, and enter the **AWS ID** from the {{site.konnect_short_name}} UI (**API Gateway > Networks**).
    1. Create the resource share and save the resulting **RAM Share ARN**.

    {:.warning}

>     > **Limitation for multi-network environments:** The same Transit Gateway cannot be used in multiple networks, even though it is shared via different RAM shares. You must create a Transit Gateway for each Dedicated Cloud Gateway network in your deployment.

Author: ```suggestion
    > **Limitation for multi-network environments:** The same Transit Gateway can be used in multiple networks, but they must be shared by different RAM shares.
```
